content/en/articles/tdd/index.md

```diff
@@ -0,0 +1,294 @@
+---
+date: "2017-01-07"  
+title: "Getting started with TDD"  
+tags:
+  - tdd
+  - bdd
+  - ci  
+authors:
+  - Freezystem
+---
+
+## Introduction
+
+Everything begins in 1999, when [Kent Beck](https://en.wikipedia.org/wiki/Kent_Beck) 
+present a new software development methodology based on AGILE: the eXtreme Programmming aka _XP_.  
+_XP_ define new optimized development practices improving code production and strength.  
```

Comment:
sinon je mets `... optimized development practices **that** improv**e** code ...`